R/flex_dashboard.R

```diff
@@ -191,18 +191,19 @@ flex_dashboard <- function(fig_width = 6.0,
     opts_orig <- c(opts_orig, options(DT.fillContainer = TRUE))
   }
 
-  # add hook to capture fig.width and fig.height and serialized
-  # them into the DOM
+  # add hook to capture fig.width and fig.height and serialize them into the DOM
   figSizePixels <- function(size) as.integer(size * 96)
+  default_knitr_chunk_hook <- knitr::hooks_markdown()[["chunk"]]
```

Comment:
It might be nice to add to the comment above about why using `knitr::hooks_markdown()` is better than stomping on it

Reply:
I tried writing out a comment with couple wording variations and it always seemed to repeat what's described in the `default_knitr_chunk_hook()` name.